test/domain/relation/explore.spec.ts

```diff
@@ -0,0 +1,56 @@
+
+import { describe, expect, it } from 'vitest';
+import { CREATOR3, CREATOR4, CREATOR5, REQUESTER1, SortOptions, explore } from './_fixtures/relation.fixture';
+
+describe('domain/relation/explore', () =>
+{
+    it('should explore relations based on popularity', async () =>
+    {
+        const relations = await explore(REQUESTER1, SortOptions.POPULAR, undefined);
+
+        expect(relations.length).toBe(3);
+        expect(relations[0].following?.id).toBe(CREATOR4);
+        expect(relations[1].following?.id).toBe(CREATOR3);
+        expect(relations[2].following?.id).toBe(CREATOR5);
+        expect(relations[0].follower).toBeUndefined();
+    });
+
+    it('should explore relations based on joindAt date', async () =>
+    {
+        const relations = await explore(REQUESTER1, SortOptions.RECENT, undefined);
+
+        expect(relations.length).toBe(3);
+        expect(relations[0].following?.id).toBe(CREATOR3);
+        expect(relations[1].following?.id).toBe(CREATOR5);
+        expect(relations[2].following?.id).toBe(CREATOR4);
+    });
+
+    it('should find no relations based on search', async () =>
+    {
+        const relations = await explore(REQUESTER1, SortOptions.POPULAR, 'bax');
+        expect(relations[0]).toBeUndefined();
+    });
+
+    it('should explore relations based of fullname search hit', async () =>
```

Comment:
This description is inconsistent with the search description above (which I personally find more descriptive). This also goes for the descriptions below.